posthog/api/test/base.py

```diff
@@ -21,7 +20,6 @@ def _create_user(self, email: str, password: Optional[str] = None, first_name: s
 
     def setUp(self):
         super().setUp()  # type: ignore
-        clear_cache()
```

Comment:
This made sense for clickhouse tests when we were storing persistent information about captured unique elements inside redis. It seems like that's no longer being used, so probably fine to remove.

Reply:
Yup, all dead code. :+1: